korgw/src/commonMain/kotlin/com/soywiz/korag/OpenglAG.kt

```diff
@@ -207,7 +207,7 @@ abstract class AGOpengl : AG() {
         val currentRenderBuffer = this.currentRenderBuffer ?: return
         if (currentRenderBuffer === mainRenderBuffer) {
             var realScissors: Rectangle? = finalScissorBL
-            realScissors?.setTo(0, 0, realBackWidth, realBackHeight)
+            realScissors?.setTo(0.0, 0.0, realBackWidth, realBackHeight)
```

Comment:
`realBackWidth` and `realBackHeight` are integers. Same issue described above.

Reply:
Thanks for letting me know.